src/utils/input/index.js

```diff
@@ -47,22 +44,22 @@ const getInputsFromSources = async (
 
 	const passphrase =
 		typeof stdIn.passphrase !== 'string' && passphraseInput
-			? await getPassphrase(vorpal, passphraseInput.source, {
+			? await getPassphrase(passphraseInput.source, {
 					shouldRepeat: passphraseInput.repeatPrompt,
 				})
 			: stdIn.passphrase || null;
 
 	const secondPassphrase =
 		typeof stdIn.secondPassphrase !== 'string' && secondPassphraseInput
-			? await getPassphrase(vorpal, secondPassphraseInput.source, {
+			? await getPassphrase(secondPassphraseInput.source, {
```

Comment:
please use parentheses for shorthand if blocks. Also please avoid using shorthand-ifs if it's longer than single line.